syft/pkg/cpe_by_specificity.go

```diff
@@ -1,20 +1,20 @@
-package cpe
+package pkg
```

Comment:
Was this moved because of the `merge` function that was added for packages?

Reply:
I updated the catalog to include path indexes as well -- good call . Commit 414acc6cecdcb5a6b2dd744f484c418178d1f6c8